### DISCUSSION TITLE
i have a question

### DISCUSSION THREAD

**Discussion by Eirxne** [community/community#67811] [Discussions]
### Select Topic Area

Question

### Body

does anyone know the puppeteer cors problem

**Comment by Atakan75** [comment]
https://stackoverflow.com/questions/52129649/puppeteer-cors-mistake
can help you

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
